x/oracle/keeper/oracle.go

```diff
@@ -12,25 +12,21 @@ import (
 func (k Keeper) RegisterOracle(ctx sdk.Context, msg *types.MsgRegisterOracle) error {
 	oracleRegistration := types.NewOracleRegistration(msg)
 
-	if err := oracleRegistration.ValidateBasic(); err != nil {
-		return err
-	}
-
```

Comment:
As [commented in the previous PR](https://github.com/medibloc/panacea-core/pull/579#discussion_r1059219257), I deleted this validation.